src/button/Button.story.mdx

```diff
@@ -0,0 +1,254 @@
+
+When user taps on it there’s a pressed state.
+After tapping if request is already loaded we either go to the next page or close the modal or if it’s a final decision we display the checked state.
+If request is not loaded we display the loading state of these buttons. When loaded, we either go to the next page or close the modal or if it’s a final decision we display the checked state.
+
+<Preview>
+  {renderPrimaryButton()}
+</Preview>
+
+> With an icon
+
+Continue buttons
+These buttons are used in the middle of the flow and never at the end of it.
+
+When user taps on it there’s a pressed state.
+After tapping if request is already loaded we go to the next page.
+If request is not loaded we display the loading state of these buttons. When loaded, we go to the next page. There’s no validated state for this component as it’s never a final decision button.
+
+<Preview>
+  <Button
+    status={ButtonStatus.PRIMARY}
+    {...commonProps}
+  >
+    <ArrowIcon right iconColor={color.white} /> Go there
+  </Button>
+</Preview>
+
+### When should I use it?
+
+> With text
+
+Use it at the end of a flow. Allows the member to make a final decision.
+
+- ✅ Use when making a final decision
+- ✅ Always centred in the page
+- ⛔️ Can’t go on 2 lines. If too long we use “…”
+- ⛔️ Not always sticky
+- ℹ️ No minimum size.
+- 🖊 Max. 20 characters in EN, max. 30 in other languages (including spaces)
+
+> With an icon
+
+Use it in the middle of a flow. Allows the member to continue to the next screen.
+
+- ✅ Always at bottom right of screen
+- ✅ Always displayed when keyboard is also displayed
+- ⛔️ Not always sticky.
+- 🖊 No text in this component.
+
+### Behaviour
+
+> With Text
+
+These buttons can either be sticky or not depending on the need.
+
+If sticky they appear above the keyboard and above other components in the page.
+
+When there’s an error that blocks the user to move forward, we hide these buttons.
+
+> With an icon
+
+These buttons are always displayed on the bottom right of the screen. They can either be sticky or not depending on the need.
+
+If sticky they appear above the keyboard and above other components in the page.
+
+When there’s an error that blocks the user to move forward, we hide these buttons.
+
+### Long texts
+
+If text is too long we can’t go on 2 lines. If too long we display “…” at the end.
+
+## Secondary button
+
+These buttons can be used as a secondary option when there’s already a primary button action.
+
+When user taps on it there’s a pressed state.
+After tapping if request is already loaded we either go to the next page or close the modal or if it’s a final decision we display the checked state then close the page (in some cases we don’t change the page like create alert in Search Results).
+If request is not loaded we display the loading state of these buttons. When loaded, we either go to the next page or close the modal or if it’s a final decision we display the checked state.
+
+<Preview>
```

Comment:
Note if you don't use `<Story name="...">` each example won't be displayed in the menu or in the canvas tab.

Reply:
I know, but right now if we put different stories in different previews, props are mixed up and everything looks the same. Tried to also add the `<Div>` I saw on slack yesterday, but it didn't work :(

Reply:
Would be good that we only have one workaround, it will be easier to spot when migrated to 6.0. And I'm totally ok to remove my `<Div />` from ConfirmationModal to do like what you did here. wdyt?

Reply:
Honestly, I'm hoping we'll upgrade soon and pass on all stories to fix these kind of things